src/Console/ModelsCommand.php

```diff
@@ -976,4 +980,18 @@ protected function checkForCustomLaravelCasts(string $type): ?string
 
         return $type;
     }
+
+    /**
+     * @param \Illuminate\Database\Eloquent\Model $model
+     */
+    protected function runModelHooks($model): void
+    {
+        $hooks = $this->laravel['config']->get('ide-helper.model_hooks', array());
+
+        foreach ($hooks as $hook) {
+            /** @var ModelHookInterface $hookInstance */
+            $hookInstance = new $hook();
```

Comment:
IMHO this should resolve the class from the container so DI can be properly used.

Reply:
Great point. I'll definitely add that. I was also thinking of checking if the created instance is an instance of the `ModelHookInterface` to guard against missing method.